evap/static/scss/_adjustments.scss

```diff
@@ -46,6 +47,12 @@ body, .fixed-top, .fixed-bottom, .is-fixed, .sticky-top {
     white-space: normal;
 }
 
+.select2-container--default .select2-selection--multiple .select2-selection__choice{
+    float: unset;
+    max-width: 90%;
+    width: fit-content;
+}
```

Comment:
I think this affect all select2 multi-selects, right? @janno42 is this what we want?

Reply:
we should only change the user select fields in this way